src/id3/v2/tag.rs

```diff
@@ -547,21 +607,54 @@ impl SplitAndMergeTag for ID3v2Tag {
 		fn split_pair(
 			content: &str,
 			tag: &mut Tag,
-			current_key: ItemKey,
+			number_key: ItemKey,
 			total_key: ItemKey,
 		) -> Option<()> {
+			fn parse_number(source: &str) -> Option<&str> {
+				let number = source.trim();
+
+				if number.is_empty() {
+					return None;
```

Comment:
Returning an empty `str` (= `Some(number)`) would be an option to distinguish between empty and invalid numbers. There are many possibilities for encoding the 3 variants _empty_, _number_, and _invalid_.